page-templates/fullwidthpage.php

```diff
@@ -16,9 +16,14 @@
 if ( is_front_page() ) {
 	get_template_part( 'global-templates/hero' );
 }
+
+$wrapper_id = 'full-width-page-wrapper';
+if ( is_page_template( 'page-templates/no-title.php' ) ) {
+	$wrapper_id = 'no-title-page-wrapper';
+}
 ?>
 
-<div class="wrapper" id="full-width-page-wrapper">
+<div class="wrapper" id="<?php echo $wrapper_id; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- ok. ?>">
```

Comment:
What's the reasoning for not just using `esc_attr()` here?

Reply:
We know that we will never have to escape the attribute. It's either `'full-width-page-wrapper'` or `'no-title-page-wrapper'`. Calling `esc_attr()` just adds a call without any benefit.